### ISSUE TITLE
Cannot acess to my private git repo. I have a password but cannot find were i can insert the password to acess the GIT repo.

### ISSUE THREAD

**Issue by voxelshaper** [zaggino/brackets-git#524]
[Please describe your problem here]

---

_Brackets 0.38.0-12606 (mac), Git 1.9.2 & Brackets-Git 0.13.15_

```
Error: Cloning into '.'...
ssh: Could not resolve hostname admin: nodename nor servname provided, or not known
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

```
call: git clone ssh://***.git . --progress
Error: Cloning into '.'...
ssh: Could not resolve hostname admin: nodename nor servname provided, or not known
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
    at Object.exports.toError (file:///Users/WhiteCat/Library/Application%20Support/Brackets/extensions/user/zaggino.brackets-git/src/ErrorHandler.js:158:19)
    at NodeConnection.<anonymous> (file:///Users/WhiteCat/Library/Application%20Support/Brackets/extensions/user/zaggino.brackets-git/src/Cli.js:177:54)
    at j (file:///Applications/Brackets.app/Contents/www/thirdparty/thirdparty.min.js:550:26676)
    at Object.k.fireWith (file:///Applications/Brackets.app/Contents/www/thirdparty/thirdparty.min.js:550:27489)
    at NodeConnection._receive (/utils/NodeConnection.js:486:34)
```

**Comment by laubsterboy** [comment]
On Windows I wasn't able to push/pull from ssh protected remote repositories until setting an environment variable `GIT_SSH` to `C:\Program Files (x86)\PuTTY\plink.exe` (provided PuTTY/plink is installed).

**Comment by Pysis868** [comment]
I followed laubsterboy's post on Jul 30, 2015, at 10:18 AM EDT, which seemed to help!  I did get stuck at another stage however, and had to fix another part of my environment.
Here are the steps I followed:

I first set the `GIT_SSH` variable according to laubsterboy's post.
Then I tested using a `git pull`, and received this error message:
```
The server's host key is not cached in the registry. You
have no guarantee that the server is the computer you
think it is.
The server's rsa2 key fingerprint is:
ssh-rsa 2048 00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00
Connection abandoned.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

I then executed this command from the terminal `$GIT_SSH git@github.com`, and entered 'y' to store this server's key.

After this I had the strangest error with Pageant finding a matching key that would not work on GitHub.  Here is the error I had so it can be searched:
```
Reading private key file "....ppk"
Pageant is running. Requesting keys.
Pageant has 3 SSH-2 keys
Pageant key #1 matches configured key file
Using username "git".
Trying Pageant key #1
Server refused our key
Disconnected: No supported authentication methods available (server sent: public
key)
FATAL ERROR: Disconnected: No supported authentication methods available (server
 sent: publickey)
```

I found with this that it was using a key file I had specified in PuTTy under the "(Default Settings)" saved session that exists from installation of that program, and plink had read this session configuration to only use that single file.  I had removed the key from Pageant in earlier testing, but it did not work when it was added then either, so it probably tried to only use that one key then as well.

Removing this key entry in the PuTTy default session configuration allowed plink to find the right key from the list in Pageant, and successfully communicate with my repository on GitHub!
